src/Stack/Stack.module.css

```diff
@@ -0,0 +1,189 @@
+.Stack {
+  display: flex;
+}
+
+/*
+ * Direction
+ */
+.Stack--vertical {
+  flex-direction: column;
+}
+
+.Stack--horizontal {
+  flex-direction: row;
+}
+
+.Stack-narrow--vertical {
+  flex-direction: column;
+}
+
+.Stack-narrow--horizontal {
+  flex-direction: row;
+}
+
+@media screen and (min-width: 48rem) {
```

Comment:
Can we use design tokens in @media queries?

Reply:
Not natively, but spoke to @langermank yesterday about this and there's a `postcss` processor that can do some compile-time substitution using our viewport tokens, which we do have available.

@langermank - are you able to share please? Alternatively, happy to put our own up. Either way, I'll likely put it in a separate PR and do a sweep of all other hardcoded @media queries in Primer Brand.